projects/components/src/header/application/application-header.component.ts

```diff
@@ -48,8 +47,8 @@ export class ApplicationHeaderComponent {
     private readonly featureStateResolver: FeatureStateResolver
   ) {
     this.pageLevelTimeRangeDisabled$ = this.featureStateResolver
-      .getFeatureState(ApplicationFeature.PageTimeRange)
-      .pipe(map(featureState => featureState === FeatureState.Disabled));
+      .getFeatureFlagValue<boolean>(ApplicationFeature.PageTimeRange)
```

Comment:
likewise here and a couple other spots in later files

Reply:
fixed